src/Microsoft.PowerShell.Commands.Utility/commands/utility/ConvertFrom-StringData.cs

```diff
@@ -30,19 +30,34 @@ public string StringData
             {
                 return _stringData;
             }
+
             set
             {
                 _stringData = value;
             }
         }
 
+        private string _Delimiter;
+
+        /// <summary>
+        /// The delimiter to index from to create the object
```

Comment:
Should starts with "Gets or sets" and has final dot.
```suggestion
        /// Gets or sets the delimiter.
```

Reply:
Addressed in most recent commit